pavelib/prereqs.py

```diff
@@ -213,6 +213,11 @@ def uninstall_python_packages():
             sh("pip uninstall --disable-pip-version-check -y django-storages")
             uninstalled = True
 
+        # Uninstall django-oauth2-provider
+        if any("django-oauth2-provider==" in line for line in frozen):
+            sh("pip uninstall --disable-pip-version-check -y django-oauth2-provider")
+            uninstalled = True
+
```

Comment:
I see you did an experiment that indicated you needed the uninstall.  I'm trying to reproduce this to understand why.  The migration to non-e installs will be difficult if each needs an uninstall like this.